src/main/java/ch/njol/skript/expressions/ExprLastDamage.java

```diff
@@ -72,35 +59,38 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 				return null;
 		}
 	}
-	
+
+	@SuppressWarnings("ConstantValue")
 	@Override
 	public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
-		if (delta != null) {
+		if (delta != null && delta[0] instanceof Number) {
+			double damage = ((Number) delta[0]).doubleValue() * 2;
+
 			switch (mode) {
 				case SET:
 					for (LivingEntity entity : getExpr().getArray(e))
-						entity.setLastDamage((Long) delta[0]);
+						entity.setLastDamage(damage);
 					break;
 				case REMOVE:
-					delta[0] = (Long) delta[0] * -1;
+					damage = damage * -1;
 				case ADD:
 					for (LivingEntity entity : getExpr().getArray(e))
-						entity.setLastDamage((Long) delta[0] + entity.getLastDamage());
+						entity.setLastDamage(damage + entity.getLastDamage());
```

Comment:
does this allow negative values?

Reply:
ill test shortly

Reply:
update: tested and yes negative values are allowed